cypress/e2e/immutableDatabaseTests/admin.ts

```diff
@@ -29,4 +29,11 @@ describe('Admin UI', () => {
       cy.get('[data-cy=account-is-admin]').should('exist');
     });
   });
+  describe('Userpage', () => {
+    it('Admin can view other linked accounts of a user', () => {
+      cy.visit('/users/user_A');
+      goToTab('Other Linked Accounts');
+      cy.get('[data-cy=other-linked-accounts-quay]').should('be.visible');
+    });
+  });
```

Comment:
Probably would be good to have a test that confirms that non-admin third parties _can't_ see the other linked accounts (assuming that's the desired behavior), so we don't accidentally leak the info.